components/builder-jobsrv/src/server/handlers.rs

```diff
@@ -219,12 +219,16 @@ pub fn job_group_cancel(req: &RpcMessage, state: &AppState) -> Result<RpcMessage
     RpcMessage::make(&net::NetOk::new()).map_err(Error::BuilderCore)
 }
 
-fn is_project_buildable(state: &AppState, project_name: &str, target: &str) -> bool {
+fn is_project_buildable(state: &AppState, project_name: &str, mut target: &str) -> bool {
     let conn = match state.db.get_conn().map_err(Error::Db) {
         Ok(conn_ref) => conn_ref,
         Err(_) => return false,
     };
 
+    if feat::is_enabled(feat::LegacyProject) {
```

Comment:
Minor - you don't need to make the target mut, you should just be able to rebind it by:
```
let target = "x86_64-linux";
```